doc/dev/release-checklists.rst

```diff
@@ -17,7 +17,7 @@ Versions and tags
 
 - [x] Update CMakeLists.txt VERSION (right at the top to X.0.0)
 - [x] Update src/ceph_release with the new release name, number, and type ('dev')
-- [x] Initial tag vX.0.0 (so that we can distinguish from (and sort
+- [ ] Initial tag vX.0.0 (so that we can distinguish from (and sort
   after) the backported (X-1).2.Z versions.
```

Comment:
should add OSDMap::pending_require_osd_release() update

Reply:
Thanks for pointing this out! Fixed `OSDMap::pending_require_osd_release()` and added the item to the checklist.